### PR TITLE
deferred rendering in overview tab

### PR DESCRIPTION
**Pull request by Argn0**
this improves performance quite a bit.

### CODE REVIEW

File: src/components/Match/Overview/Overview.jsx [hunk 1]
```diff
@@ -6,10 +6,13 @@ import AbilityDraftTable from '../AbilityDraftTable';
...
 import BuildingMap from '../BuildingMap';
 import Collapsible from './../../Collapsible/index';
-import AbilityBuildTableSkeleton from './../../Skeletons/AbilityBuildTableSkeleton';
+import AbilityBuildTable from '../AbilityBuildTable';
+import deferredComponent from './../../DeferredComponent/index';
 
-const AbilityBuildTable = React.lazy(() =>
-  import(/* webpackChunkName: 'AbilityBuildTable' */ '../AbilityBuildTable'));
+
+const DeferredAbilityBuildTable = deferredComponent(AbilityBuildTable);
```
Comment (howardchung) [inline src/components/Match/Overview/Overview.jsx]: Any reason we can't just make this a JSX component?

Comment (Argn0) [reply src/components/Match/Overview/Overview.jsx]: There is no particular reason. I guess we could use a render prop.

Comment (howardchung) [reply src/components/Match/Overview/Overview.jsx]: I think I prefer that pattern a little more.

Comment (howardchung) [reply src/components/Match/Overview/Overview.jsx]: or even just this.props.children?

**Comment by Argn0** [comment]
Tested on this page https://deploy-preview-2294--opendota.netlify.com/matches/5117971814/overview

going from the Benchmarks to Overview tab is measured ~300ms faster for me.

**Comment by Argn0** [comment]
Also it seems like the biggest performance hit seems to come from the number of icons in the Ability Build tables.

**Pull request merged**